src/lib.rs

```diff
@@ -146,31 +146,31 @@
 
 // Compile with `no_std` when the `std` feature is not specified.
 #![cfg_attr(not(feature = "std"), no_std)]
-
```

Comment:
I would have liked to maintain these blank lines, but that doesn't seem possible.